interval/relationship.go

```diff
@@ -126,3 +128,32 @@ func PrintRelationships() {
 	fmt.Println("\"end\"    = f + fi + e")
 	fmt.Println("\"equal\"  = e")
 }
+
+//matchOverlapLen returns the number of bases shared between two start and end points. Used in MatchProportion.
+func matchOverlapLen(start1 int, end1 int, start2 int, end2 int) int {
+	return numbers.Max(0, numbers.Min(end1, end2)-numbers.Max(start1, start2))
+}
+
+//MatchProportion returns the proportion of bases in the target and query that can be lifted for a particular interval as a pair of floats (propT, propQ)
```

Comment:
Should this be in the new lift package?

Reply:
Only uses intervals, could go either way

Reply:
It does seem like this should be in lift, but maybe that causes a circular dependency to have interval in lift (is lift already in interval?)?